### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/15/day14.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#24] [Comments]
# https://mnfienen.github.io/talulat/2023/08/15/day14.html

Day 14 - Some days go late miles today:83 total miles: 860 km today: 134 total km: 1386

https://mnfienen.github.io/talulat/2023/08/15/day14.html

**Comment by aleaf** [comment]
that eggs benny looks so good!

**Reply by mnfienen** [reply]
so good!
